package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "checkdotenv",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
@gabrielrinaldi We discussed if It will better a patch increase or a major one and as we are kinda of recreating that repo we choose to start in a major `2.0.0`, in fact any special reason.